test/index.test.js

```diff
@@ -72,23 +72,23 @@ const mocks = {
 
                 res.end();
                 if (hitCount >= 2) {
-                    server.close(done);
+                    server.close();
                 }
             });
         });
 
         return server;
     },
 
-    getUdpServer(expectedNumberOfEvents, expectedMsg, done) {
+    getUdpServer (expectedNumberOfEvents, expectedMsg) {
         let hitCount = 0;
         const server = Dgram.createSocket('udp4');
         server.on('message', (msg) => {
             hitCount += 1;
             validateResponses(msg.toString(), expectedNumberOfEvents, expectedMsg);
 
             if (hitCount >= 2) {
-                server.close(done);
+                server.close();
```

Comment:
Two test failures in the Travis build. Perhaps you still need to pass a callback?

Reply:
Yes, the methods still need `async`/`await` calls, but my 4:30 AM self had problems getting it running. :grinning: – That's basically why I labeled it "WIP".